platform/ui/src/gatsby-theme-docz/tailwind.css

```diff
@@ -22,3 +22,14 @@
 .ohif-scrollbar::-webkit-scrollbar-thumb:window-inactive {
   @apply bg-primary-main;
 }
+
+/* TOOLTIP WORKAROUND FOR ARROW UP */
+.tooltip::before {
+  @apply bg-primary-dark absolute z-10;
+  content: '';
+  width: 14px;
+  height: 1px;
+  top: -1px;
+  left: 50%;
+  transform: translateX(-50%);
+}
```

Comment:
This is the gatsby theme or our theme?

Reply:
yeah, some very specific custom styles are here. Better split?

Reply:
Eventually. Fine to punt for now.

Reply:
I've created a separate CSS to use when we need to create custom classes for the components.